cmake/modules/RootCTestMacros.cmake

```diff
@@ -131,24 +129,28 @@ function(ROOTTEST_ADD_TEST testname)
   endif()
 
   if (ARG_ENABLE_IF OR ARG_DISABLE_IF)
-    ROOT_SHOW_OPTIONS(ROOT_ENABLED_FEATURES)
     # Turn the output into a cmake list which is easier to work with.
-    STRING(REPLACE " " ";" ROOT_ENABLED_FEATURES "${ROOT_ENABLED_FEATURES}")
+    set(ROOT_ENABLED_FEATURES ${_root_enabled_options})
+    set(ROOT_ALL_FEATURES ${_root_all_options})
```

Comment:
This variable is not set anywhere that I can see, I checked root.git,  roottest.git, and installed files from ROOT.

Reply:
This variable is part of PR root-project/root#2122